.github/workflows/build-web.yml

```diff
@@ -44,8 +44,8 @@ jobs:
       - name: Install dependencies
         run: poetry install
       - name: Build Web Site
-        # run: poetry run build-mappings-explorer --url-prefix 'https://mappingsexplorer.z13.web.core.windows.net/${{env.BRANCH_NAME}}/'
-        run: poetry run build-mappings-explorer --url-prefix 'https://center-for-threat-informed-defense.github.io/mappings-explorer/'
+        run: poetry run build-mappings-explorer --url-prefix 'https://mappingsexplorer.z13.web.core.windows.net/${{env.BRANCH_NAME}}/'
+        # run: poetry run build-mappings-explorer --url-prefix 'https://center-for-threat-informed-defense.github.io/mappings-explorer/'
```

Comment:
I think this build path is for production, right? Why does it have the preview URL prefix?